docs/user_manual/C/gtg-caldav.page

```diff
@@ -26,9 +26,11 @@
 
   </info>
 
-  <title>CalDAV Sync Service</title>
+  <title>CalDAV: Calendar Sync Service</title>
 
-  <p>CalDAV is a calendaring protocol that allows a client to access items from a server. Sync your third-party applications that support the CalDAV standard with <app>GTG</app> to manage all of your tasks and calendar items in one place. The options and required fields for the CalDAV Sync Service are described below. Once you have filled in all of the required information, click <em style="strong">Enable syncing</em> to start importing your CalDAV items.</p> 
+  <p>CalDAV is a calendaring protocol that allows a client to access items from a server. CalDAV is often provided by calendar and task/groupware services. Sync your third-party applications that support the CalDAV standard with <app>GTG</app> to manage all of your tasks and calendar items in one place. The options and required fields for the CalDAV Sync Service are described below. Once you have filled in all of the required information, click <em style="strong">Enable syncing</em> to start importing your CalDAV items.</p> 
```

Comment:
I think the wording got a bit weird here with the groupware addition, like you say it is often provided by groupware services, and then "Sync your third-party applications that support the CalDAV standard with GTG to manage all of your tasks and calendar items in one place.". 

My try:

```suggestion
  <p>CalDAV is a calendaring protocol that allows a client to access items from a server. Many calendar and task services (often included in groupware packages) provide this feature. <app>GTG</app> supports for this standard to be able to manage all your tasks in one place. The options and required fields for the CalDAV Sync Service are described below. Once you have filled in all of the required information, click <em style="strong">Enable syncing</em> to start importing your CalDAV items.</p> 
```